frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingSdkSettings.tsx

```diff
@@ -14,37 +18,108 @@ import { SettingTextInput } from "../widgets/SettingTextInput";
 import { useMergeSetting } from "./hooks";
 import type { AdminSettingComponentProps } from "./types";
 
-const SDK_ORIGINS_SETTING = {
-  key: "embedding-app-origins-sdk",
-  placeholder: "https://*.example.com",
-  display_name: t`Cross-Origin Resource Sharing (CORS)`,
-  description: jt`Enter the origins for the websites or web apps where you want to allow SDK embedding, separated by a space. Here are the ${(
-    <ExternalLink
-      key="specs"
-      href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors"
-    >
-      {t`exact specifications`}
-    </ExternalLink>
-  )} for what can be entered.`,
-} as const;
-
 export function EmbeddingSdkSettings({
   updateSetting,
 }: AdminSettingComponentProps) {
-  const sdkOriginsSetting = useMergeSetting(SDK_ORIGINS_SETTING);
-
+  const isEE = PLUGIN_EMBEDDING_SDK.isEnabled();
   const isEmbeddingSdkEnabled = useSetting("enable-embedding-sdk");
+  const canEditSdkOrigins = isEE && isEmbeddingSdkEnabled;
+
+  const isHosted = useSetting("is-hosted?");
+
+  const upgradeUrl = useSelector(state =>
+    getUpgradeUrl(state, {
+      utm_campaign: "embedding-sdk",
+      utm_content: "embedding-sdk-admin",
+    }),
+  );
+
+  const sdkOriginsSetting = useMergeSetting(
+    !isEE
+      ? {
+          key: "embedding-app-origins-sdk",
+          placeholder: "https://*.example.com",
+          display_name: t`Cross-Origin Resource Sharing (CORS)`,
+          description: jt`Try out the SDK on localhost. To enable other sites, ${(
+            <ExternalLink key="upgrade-url" href={upgradeUrl}>
+              {t`upgrade to Metabase Pro`}
+            </ExternalLink>
+          )} and Enter the origins for the websites or apps where you want to allow SDK embedding.`,
+        }
+      : {
+          key: "embedding-app-origins-sdk",
+          placeholder: "https://*.example.com",
+          display_name: t`Cross-Origin Resource Sharing (CORS)`,
+          description: t`Enter the origins for the websites or apps where you want to allow SDK embedding, separated by a space. Localhost is automatically included.`,
+        },
+  );
 
   function handleChangeSdkOrigins(value: string | null) {
     updateSetting({ key: sdkOriginsSetting.key }, value);
   }
-  const hasEmbeddingSdkFeature = PLUGIN_EMBEDDING_SDK.isEnabled();
-  const canEditSdkOrigins = hasEmbeddingSdkFeature && isEmbeddingSdkEnabled;
 
   function handleToggleEmbeddingSdk(event: ChangeEvent<HTMLInputElement>) {
     updateSetting({ key: "enable-embedding-sdk" }, event.target.checked);
   }
 
+  const switchMetabaseBinariesUrl = useDocsUrl(
+    "https://www.metabase.com/docs/latest/paid-features/activating-the-enterprise-edition",
```

Comment:
This useDocsUrl is local to the file, we already have a [`useDocsUrl` function](https://github.com/metabase/metabase/blob/3b533c49ff03d94dc74abf922ac5dde5c779165f/frontend/src/metabase/common/hooks/use-docs-url/use-docs-url.ts#L5) that links version of the docs that's the one used by the instance, is the use of "latest" a deliberate product requirement?

@albertoperdomo maybe you can answer this

Reply:
Judging by how the link is used, I don't see a reason why we aren't using the common useDocsUrl hook. This function just adds the UTM parameters (which makes me think I should rename the function, which I will)

Reply:
I'd propose we add such new UTM tags to the shared `useDocsUrl`. The reason I used the local one was because the shared one doesn't have the new UTM tags.